spec/Dynamic-Validator.md

```diff
@@ -0,0 +1,144 @@
+# Dynamic Validator
+
+## Constatns
+| name                             | value         |
+|----------------------------------|---------------|
+| **TERM**                         | 1 hour        |
+| **NOMINATION_EXPIRATION**        | 24 **TERM**s  |
+| **CUSTODY_PERIOD**               | 24 **TERM**s  |
+| **RELEASE_PERIOD**(TBD)          | 240 **TERM**s |
+| **WITHDRAW_DELAY**               | 1 **TERM**    |
+| **MAX_NUM_OF_VALIDATORS**        | 30            |
+| **MIN_NUM_OF_VALIDATORS**        | TBD(may be 4) |
+| **MIN_CCS_RATE_TO_BE_VALIDATOR** | 0.01          |
+| **MIN_DEPOSIT**                  | TBD CCC       |
+
+
+## FSM of Account States
+```
+                                         +--------+  
+                              /--------->| Banned |<---+-------\
```

Comment:
There is no way out of banned state, right?

Reply:
Yes, there is no way.